modules/graphics/enums/VertexWinding.lua

```diff
@@ -0,0 +1,14 @@
+return {
```

Comment:
The enum has to be required in the module's enum list:

https://github.com/love2d-community/love-api/blob/master/modules/graphics/Graphics.lua#L3540